python/src/aiconfig/Config.py

```diff
@@ -53,11 +52,14 @@
 ModelParserRegistry.register_model_parser(
     DefaultAnyscaleEndpointParser("AnyscaleEndpoint")
 )
+ModelParserRegistry.register_model_parser(
+    # TODO: Get a list of default models for AzureOpenAIParser
+    AzureOpenAIParser(deployment="gpt-35-turbo")
```

Comment:
nit, maybe a comment explaining 35 is intentional (not 3.5) based on https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models#gpt-35

Reply:
I don't think we should not add this as a default model parser because that deployment id is specific to LastMile